ci-operator/step-registry/aws/provision/localzone-subnet/aws-provision-localzone-subnet-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: aws-provision-localzone-subnet
```

Comment:
The same as this one, since the `aws-provision-localzone-subnet` chain and `aws-provision-localzone-subnet` step actually also configure `wavelength-zone` settings, maybe we can rename the chain and step as `aws-provision-edgezone-subnet`?